bookmarks/services/website_loader.py

```diff
@@ -29,9 +29,9 @@ def load_website_metadata(url: str):
         page_text = load_page(url)
         soup = BeautifulSoup(page_text, 'html.parser')
 
-        title = soup.title.string if soup.title is not None else None
+        title = soup.title.string.strip() if soup.title is not None else None
         description_tag = soup.find('meta', attrs={'name': 'description'})
-        description = description_tag['content'] if description_tag is not None else None
+        description = description_tag['content'].strip() if description_tag is not None else None
```

Comment:
```suggestion
        description = description_tag['content'].strip() if description_tag and description_tag['content'] else None
```
I think this could result in a null reference if the `content` attribute does not exist.

Reply:
Done